core/util/runChromy.js

```diff
@@ -247,30 +149,15 @@ function processScenarioView (scenario, variantOrScenarioLabelSafe, scenarioLabe
   }
   chromy.goto(url);
 
-// =================UPDATE vvvv ==================
-  // --- load in backstopTools into client app ---
-  // chromy.inject('js', config.env.backstop + BACKSTOP_TOOLS_PATH);
-  chromy.defineFunction(exists);
-  chromy.defineFunction(isVisible);
-  chromy.defineFunction(expandSelectors);
-  chromy.defineFunction(startConsoleLogger);
-  chromy.defineFunction(hasLogged);
-  chromy.evaluate(_ => startConsoleLogger());
-  // console.log('Loading >>>' + config.env.backstop + BACKSTOP_TOOLS_PATH);
-  // chromy.inject('js', config.env.backstop + BACKSTOP_TOOLS_PATH);
-
-  // chromy.evaluate(() => console.log(exists))
-  // chromy.evaluate(() => console.log("window.exists ^^^^"))
-
-// =================UPDATE ^^^^^^^^^ ==================
+  injectBackstopTools(chromy);
```

Comment:
I've tried to do it conditionally but it failed then.

````javascript
chromy.evaluate(() => {
    return !window._backstopTools;
}).result(_result => {
    if (_result) {
      injectBackstopTools(chromy)
    }
});
````
I don't have time left today to figure this out.